pkg/pillar/cmd/zedagent/zedagent.go

```diff
@@ -1791,7 +1801,18 @@ func handleDPCLImpl(ctxArg interface{}, key string,
 		log.Functionf("handleDPCLImpl: ignoring %s", key)
 		return
 	}
-	triggerPublishDevInfo(ctx)
+
+	status := statusArg.(types.DevicePortConfigList)
+	var exclude []string
+	exclude = append(exclude, "LastFailed:")
```

Comment:
Don't we want to see changes to LastFailed?

Reply:
I was thinking when first time it failed, there is some 'real change' in the struct, so we'll publish right away. but later on it always fail the same way, only the timestamp change, we don't care. but the 10 min periodical publish will reflect the 'LastFailed' correctly since it gets the most recent one.

Reply:
ok